docs/index.md

```diff
@@ -28,8 +28,9 @@ Also [John MacFarlane](https://github.com/jgm) for his work on the CommonMark sp
 - <http://talk.commonmark.org> - CommonMark forum, good place to collaborate developers' efforts.
 
 ```{toctree}
-:maxdepth: 2
-
+---
```

Comment:
It would be nice if this did not convert from the short-hand

Reply:
but mainly looking good cheers!

Reply:
> It would be nice if this did not convert from the short-hand

Hmm, maybe. I couldn't see any crystal clear advantage to using the short-hand, besides maybe being one line less verbose. Always formatting with the frontmatter style has a few advantages:

- Kills useless debate in PRs about which style to use
- Removes the chance that you'll ever need to look at a frontmatter style to short-hand style diff in a PR --> cleaner diffs, simpler reviews.

> but mainly looking good cheers!

:partying_face:

Reply:
> I couldn't see any crystal clear advantage to using the short-hand

Well its more comparable to the RST syntax (to ease migration), but mainly I fear (like the consecutive numbering discussion) it may put users off using mdformat, if it enforces a style they are not used to (a lot of current documentation uses the shorthand), similarly I would envisage it as a flag

Reply:
Consecutive numbering was different to me in that that style clearly is better in its own way (more readable, and resembles rendered doc more). I don't currently feel that way about the shorthand syntax.

I definitely understand the point about putting users off, but a) mdformat is an opinionated formatter, not a text passthrough function, and b) users don't pay me so I don't necessarily have to care :smile:.

If someone were to contribute a `--keep-directive-style` flag I'd merge it.